### PR TITLE
DS-107 Ad configuration

### PR DESCRIPTION
**Pull request by walsh9**
Adds a configurable network code and ad unit prefix.

This breaks compatibility with previous versions, so we should bump the version number with the next release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,28 @@ Things you might want to know when using this addon:
...
   * You might want to use this if you're using key/value targeting, and never want a targeted ad to stick around (due to lack of other ads) after you change the targeting.
 
+## Configuration
+
+You can configure the following settings in your app's `environment.js`.
+```js
+module.exports = function(environment) {
+  let ENV = {
+    //...
+    nyprAds: {
```
Comment (noslouch) [inline README.md]: I've seen other addons use the `'dasherized-name'` as the toplevel config key, e.g. `ember-cli-babel`. What do you think?

File: addon/defaults.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const DEFAULT_NETWORK_CODE = '6483581'
```
Comment (noslouch) [inline addon/defaults.js]: Can you double check that you're getting the expected value for `DEFAULT_NETWORK_CODE` with this syntax? This is a named export, i.e. not using the `default` keyword, which I would expect to be imported like this:
```js
import { DEFAULT_NETWORK_CODE } from 'nypr-ads/defaults';
```

File: addon/components/expandable-responsive-leaderboard.js [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+    return `/${this.get('network')}${this.get('prefix')}/leaderboard/${this.get('site')}_${this.get('unitSuffix')}`;
+  }),
+  init() {
+    this._super(...arguments);
+    if (config.nyprAds) {
+      this.set('network', config.nyprAds.networkCode || DEFAULT_NETWORK_CODE);
+      this.set('prefix', config.nyprAds.prefix ? `/${config.nyprAds.prefix}` : null);
+    } else {
+      this.set('network', DEFAULT_NETWORK_CODE);
+    }
```
Comment (noslouch) [inline addon/components/expandable-responsive-leaderboard.js]: these are all build-time values, so you don't need to set them in the init. they can be set directly as class variables with the same one-line booleans.

File: README.md (hunk 1 above)
Comment (walsh9) [reply README.md]: I was trying to find examples earlier and coming up blank, but if hyphens are good enough for `ember-cli-babel` and `ember-cli-mirage` they're good enough for me.

File: addon/defaults.js (hunk 2 above)
Comment (walsh9) [reply addon/defaults.js]: Ah, good catch.

File: addon/components/expandable-responsive-leaderboard.js (hunk 3 above)
Comment (walsh9) [reply addon/components/expandable-responsive-leaderboard.js]: The linter threw warnings about leaking state when i tried this. I know it should all resolve to strings but I couldn't make the linter see things my way.

**Pull request merged**